### PR TITLE
Add validations

### PR DESCRIPTION
**Pull request by mambetakunova**

### CODE REVIEW

File: validations/index.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+export function validateId(userId: string) {
+    if (!userId) {
```
Comment (kadishmal) [inline validations/index.ts:2]: Indentation is wrong.

File: validations/test/index.test.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { validateForPositiveNumber } from '..';
+
+describe('validateForPositiveNumber', () => {
+  it('should throw exception if prop is below zero', async () => {
```
Comment (kadishmal) [inline validations/test/index.test.ts:4]: `if prop` => `if value`

File: validations/test/index.test.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    expect.assertions(1);
+
+    try {
+      await validateForPositiveNumber(-50);
+    } catch (err) {
+      expect(err).toEqual(error);
+    }
+  });
+
+  it('should throw exception if prop is not number', async () => {
```
Comment (kadishmal) [inline validations/test/index.test.ts:16]: Here as well.

File: validations/.gitignore [hunk 4]
```diff
@@ -0,0 +1,5 @@
+*.js
```
Comment (kadishmal) [inline validations/.gitignore:1]: The repository structure should be similar to that of [`react-components`](https://github.com/mancho-devs/react-components) repository.

**Review by kadishmal — CHANGES_REQUESTED** [review]

File: validations/index.ts (hunk 1 above)
Comment (mambetakunova) [reply validations/index.ts]: done

File: validations/test/index.test.ts (hunk 2 above)
Comment (mambetakunova) [reply validations/test/index.test.ts]: done

File: validations/test/index.test.ts (hunk 3 above)
Comment (mambetakunova) [reply validations/test/index.test.ts]: done

File: validations/.gitignore (hunk 4 above)
Comment (mambetakunova) [reply validations/.gitignore]: done

**Review by kadishmal — APPROVED** [review]
Сонун! 👍

**Pull request merged**